### ISSUE TITLE
Calyptos: Automatic vpc creation after install

### ISSUE THREAD

**Issue by dmccue** [eucalyptus/calyptos#66]
Can the contents of this script be added to calyptos provision and/or chef recipes so that they're left with the default vpc after deployment.
post-script-url: http://git.qa1.eucalyptus-systems.com/qa-repos/eucalele/raw/master/deploy-helpers/scripts/midonet_post.sh

git.qa1.eucalyptus-systems.com is inaccessible outside of openvpn connection

midonet_post.sh

``` bash
service eucanetd restart
source /root/eucarc
euca-create-vpc `euare-accountlist | grep '^eucalyptus' | awk '{print $2}'`
```

**Comment by viglesiasce** [comment]
Fixed here:
https://github.com/eucalyptus/eucalyptus-cookbook/pull/205

**Comment by dmccue** [comment]
Verified fixed, thanks!: https://eucalyptus.atlassian.net/secure/attachment/25611/calyptos-1441138161.tgz

```
[root@odc-f-09 ~]# euca-describe-vpcs
VPC vpc-7e0cb490    available   172.31.0.0/16   dopt-fa23585a   default true
[root@odc-f-09 ~]# euca-describe-subnets
SUBNET  subnet-7aea6bd4 available   vpc-7e0cb490    172.31.0.0/20   4091    az-01   true    true
```
